packager/hls/base/master_playlist.cc

```diff
@@ -441,6 +442,10 @@ void AppendPlaylists(const std::string& default_audio_language,
     }
   }
 
+  if (segment_sap_aligned) {
+    content->append("\n#EXT-X-INDEPENDENT-SEGMENTS\n");
+  }
+    
```

Comment:
Is there a reason that you choose to put it here? (which is between two BuildStreamInfTag..)

Is there a requirement on the order of this tag?

Reply:
It was put here based on a previous comment from #564. From trial and error, there does not seem to be a requirement on the order, but it seems most intuitive to put it closer to the beginning of the m3u8 file.